src/index.ts

```diff
@@ -16,4 +16,5 @@ app.registerInitializer({
 
 app.boot();
 
+containerElement.innerHTML = '';
```

Comment:
I'm not sure why that is the preferred way of doing this but I agree it makes sense to do this the same way FastBoot does it here as well - fixed.